db/python/layers/sequencing_group.py

```diff
@@ -160,6 +163,51 @@ async def get_type_numbers_for_project(self, project: ProjectId) -> dict[str, in
         """Get sequencing type numbers (of groups) for a project"""
         return await self.seqgt.get_type_numbers_for_project(project)
 
+    async def get_type_numbers_for_project_history(
```

Comment:
I know this method name is based on the one above, but I think it is not very clear what it is doing from the name, can I suggest something like. `get_sequencing_group_counts_by_month`